translations/en-us.yml

```diff
@@ -1391,7 +1391,7 @@ preprints:
                 base: 'This {documentType}'
     header:
         osf_preprints: 'OSF Preprints'
-        search_placeholder: 'Search {placeholder} ...'
```

Comment:
It looks like this is used in the `BrandedHeader` component, so this translation most likely should know what the provider's preprintWord is. Was there a problem that this translation string was causing?